lib/drivers/medtronic600/medtronic600Driver.js

```diff
@@ -1887,6 +1801,7 @@ module.exports = (config) => {
         .then(() => debug('Read pump history'))
         .then(() => {
           const threeMonthsAgo = new Date(new Date().valueOf() - 7776e6);
+          // const threeMonthsAgo = new Date(new Date().valueOf() - 181e6); // Three weeks
```

Comment:
I will remove it. I have it here because I'm in the middle of fixing other bugs, but upon further reflection, I'll make a Stash, rather than commented code.